README.md

```diff
@@ -32,7 +32,7 @@ You don't need to care about passing Scala version and Scalac options to this pl
 In order to execute semantic rules (e.g., `RemoveUnused`), you need to enable SemanticDB.
 
 <details>
-<summary>For Scala 2, you should add compiler plugin.</summary>
+<summary>For Scala 2.13, you should add compiler plugin. For Scala 2.13, just repalce compiler flag from "-Wunused" to "-Ywarn-unused" </summary>
```

Comment:
it's easier to write proper arg in xml, instead of writing wrong arg in xml and guide how to replace it to proper value

Reply:
okay